src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -439,8 +439,9 @@ public static string SheetsNotFound(List<string> listOfSheetNames)
             return sb.ToString();
          }
 
-         public static readonly string UseFiltersForImport = "Use the filters for importing the data";
-         
+         public static readonly string UseFiltersForImport = "Use filters for importing data";
+         public static readonly string UseFiltersForImportTooltip = "When checked, the data comming from the file will also be filter by the importer. When unchecked, the filters only affect this view";
```

Comment:
"When checked, filters applied in the preview will also be applied to imported data. When unchecked, the filters only affect this view"

It took me some time to realize that there are filters in the preview-table (and I am not sure many users know of the existence of that filters).

Reply:
>It took me some time to realize that there are filters in the preview-table (and I am not sure many users know of the existence of that filters).

After so many years with PK-Sim/MoBi? lol
There is a good guide by DevExpress for Filters: https://docs.devexpress.com/WindowsForms/114645/controls-and-libraries/data-grid/getting-started/walkthroughs/filter-and-search/tutorial-filter-editor
It's for developers, but the first half of this page describes it nicely from the user perspective - I think we should link it in the documentation.
We could also add the link to the tooltip of the "Use filters..." checkbox.

Reply:
P.S. OSP docu gives also some info on it:
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/features-of-tables#filtering